user.go

```diff
@@ -4,11 +4,29 @@ import (
 	"context"
 )
 
+// UserStatus indicates whether a user is active or inactive
+type UserStatus string
+
+// Valid validates user status
+func (u *UserStatus) Valid() error {
+	if *u != "active" && *u != "inactive" {
+		return &Error{Code: EInvalid, Msg: "Invalid user status"}
+	}
+
+	return nil
+}
+
 // User is a user. 🎉
 type User struct {
-	ID      ID     `json:"id,omitempty"`
-	Name    string `json:"name"`
-	OAuthID string `json:"oauthID,omitempty"`
+	ID      ID         `json:"id,omitempty"`
+	Name    string     `json:"name"`
+	OAuthID string     `json:"oauthID,omitempty"`
+	Status  UserStatus `json:"status,omitempty"`
```

Comment:
shall we use the influxdb.Status instead of creating new status?

Reply:
👍